sentry/hooks.py

```diff
@@ -16,17 +16,11 @@
 
 # include js, css files in header of desk.html
 # app_include_css = "/assets/sentry/css/sentry.css"
-app_include_js = [
-	"https://browser.sentry-cdn.com/5.12.1/bundle.min.js",
```

Comment:
Where are you picking up the sentry bundle?
I see you have a sentry.min.js but this PR doesn't show where this is. Or am I missing something?

Reply:
Oh, added it to the env using yarn, it's in the package.json